awswrangler/timestream.py

```diff
@@ -208,9 +224,17 @@ def write(
     >>> assert len(rejected_records) == 0
 
     """
-    measure_type: str = _data_types.timestream_type_from_pandas(df[[measure_col]])
-    _logger.debug("measure_type: %s", measure_type)
-    cols_names: List[str] = [time_col, measure_col] + dimensions_cols
+    if not measure_col and not measure_cols:
+        raise exceptions.InvalidArgumentCombination("Either `measure_col` or `measure_cols` is required.")
+    if not dimensions_cols:
+        raise exceptions.InvalidArgumentValue("`dimensions_cols` is required.")
+    measure_cols_names: List[str] = measure_cols if measure_cols else [measure_col] if measure_col else []
```

Comment:
Can't it be simplified to: `measure_cols if measure_cols else [measure_col]`? L228 ensures that at least one of `measure_col` or `measure_cols` is passed. So perhaps we don't need the `if measure_col else []`?

Reply:
For me actual code trumps type checking. If the check is not smart enough to detect that it's fine because of L228, then the issue lies with the type checking tool. So perhaps we should just add # type: ignore?

Reply:
What type checking says makes sense here, but I'm OK with ignore, too.